python/ray/dashboard/client/src/api.ts

```diff
@@ -187,21 +186,21 @@ export type FullActorInfo = {
 };
 
 export type PartialActorInfo = {
-  actorId: string;
-  actorTitle: string;
+  actorId?: string;
```

Comment:
When are they supposed to be null?

Reply:
They seem to be null all of the time for these `PartialActorInfo` which are actually pending Actor tasks. I've been relying on the mapping that the backend passes from actorId to actor info for `PartialActorInfo` instances.

Reply:
I see. I think it is probably possible to fix it (in a way we can show Pending actors' ID. This should be possible now after we enabled GCS actor management). For the scope of this PR it LGTM.